mollie/api/objects/payment.py

```diff
@@ -146,3 +147,43 @@ def settlement_amount(self):
     @property
     def method(self):
         return self._get_property('method')
+
+    @property
+    def value(self):
```

Comment:
waarom dit? is `amount` alleen niet al voldoende? value en currency zijn niet gedocumenteerd, amount wel.